src/components/modules/toolbar/blockSettings.ts

```diff
@@ -7,7 +7,9 @@ import { I18nInternalNS } from '../../i18n/namespace-internal';
 import Flipper from '../../flipper';
 import { TunesMenuConfigItem } from '../../../../types/tools';
 import { resolveAliases } from '../../utils/resolve-aliases';
-import Popover, { PopoverEvent } from '../../utils/popover';
+import { Popover, PopoverDesktop, PopoverMobile } from '../../utils/popover';
+import { PopoverEvent } from '../../utils/popover/popover.typings';
```

Comment:
use `import type` when only types are imported
```suggestion
import { type Popover, PopoverDesktop, PopoverMobile } from '../../utils/popover';
import type { PopoverEvent } from '../../utils/popover/popover.typings';
```